api/admin/oauth.py

```diff
@@ -46,7 +47,7 @@ def callback(self, request={}):
             ), redirect_url
 
     def google_error_problem_detail(self, error):
-        error_detail = GOOGLE_OAUTH_FAILURE.detail + " Error: " + error
+        error_detail = GOOGLE_OAUTH_FAILURE.detail + _("Error: ") + error
```

Comment:
I think the rule is that this should be "Error: %(error)s" so translators can vary the placement of the variable.